tests/integration/test_schema.py

```diff
@@ -36,3 +36,15 @@ def test_unnoised_id_cols(dataset_name: str, request):
         .all()
         .all()
     )
+
+
+def test_census_order(noised_sample_data_decennial_census):
+    """
+    Tests that the census gets ordered by year and household_id. Note this test will need to
+    be updated once we properly move the sorting to the guardian duplication noise function.
```

Comment:
Why should the sorting happen in the duplicate-with-guardian function? If it happens there, then wouldn't the rows be ordered differently depending on whether the user has that noise type set to 0 or not? It seems like this ordering of the rows should be a property of the dataset, not which types of noise are applied.